### PR TITLE
Update metadata for Learn docs: Bookings & Device and app management

### PR DESCRIPTION
**Pull request by Linda-Editor**
Meta titles and descriptions; other edits as needed

### CODE REVIEW

File: concepts/managedtenants-concept-overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Microsoft 365 Lighthouse API overview"
-description: "Microsoft 365 Lighthouse is an admin portal that helps Managed Service Providers (MSPs) secure and manage devices, data, and users at scale for small- and medium-sized business (SMB) customers who are using Microsoft 365 Business Premium."
+title: "Microsoft 365 Lighthouse API multi-tenant management"
```
Comment (Lauragra) [inline concepts/managedtenants-concept-overview.md:2]: This title doesn't quite make sense, the API doesn't modify multi-tenant management. Should we get Microsoft Graph into the title or search description?

File: concepts/service-communications-concept-overview.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-description: "Use the service communications API in Microsoft Graph to access the health status and message center posts about  Microsoft cloud services."
+title: "Access service health and communications through Microsoft Graph"
+description: "Use the service communications API in Microsoft Graph to access the health status and message center posts about Microsoft cloud services."
 author: "payiAzure"
 ms.localizationpriority: high
 ms.prod: "service-communications"
 ms.custom: scenarios:getting-started
 ---
 
 # Overview for accessing service health and communications in Microsoft Graph
```
Comment (Lauragra) [inline concepts/service-communications-concept-overview.md:10]: Can we remove "Overview for" here as well?

File: concepts/upload-data-to-upload-session.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ You can upload the entire file, or split the file into multiple byte ranges, as
 
 The segments of the file can be uploaded in any order and can be uploaded in parallel, with up to four concurrent requests. When all the binary segments of a document are uploaded, the binary file is linked to the **printDocument**.
 
-## HTTP request
+## Make an HTTP request
```
Comment (Lauragra) [inline concepts/upload-data-to-upload-session.md:18]: For consistency, let's just change these to ## Request and ## Response, similar to the structure of the reference topics.

File: concepts/booking-concept-overview.md [hunk 4]
```diff
@@ -1,42 +1,37 @@
...
 
-Microsoft Bookings provides online and mobile apps that make appointment scheduling simple and efficient for small businesses and their customers. Any organization that provides service on an appointment basis, such as large scale enterprise companies, auto repair shops, hair salons, and law firms, can benefit from having their bookings managed so as to free up time for the more important task to grow their business. Microsoft Bookings is available to enterprise organizations and businesses that have a Microsoft 365 Business Premium subscription.
+Microsoft Bookings provides online and mobile apps that make appointment scheduling simple and efficient for businesses and their customers. Any organization that provides service on an appointment basis, such as large scale enterprise companies, auto repair shops, hair salons, and law firms, can benefit from having their bookings managed so as to free up time for the more important task to grow their business. Microsoft Bookings is available to enterprise organizations and businesses that have a Microsoft 365 Business Premium subscription.
 
-## Why integrate with Microsoft Bookings using Microsoft Graph?
```
[the remark this answers is not in the document]
Comment (Linda-Editor) [reply concepts/booking-concept-overview.md:14]: I switched this one back, but in doing so, we just lost a lot of rich headings for the right nav and H2 crawls. :-(

File: concepts/managedtenants-concept-overview.md (hunk 1 above)
Comment (Linda-Editor) [reply concepts/managedtenants-concept-overview.md:2]: Okay. I changed the meta title and H1 to "Manage multiple customer tenants using the Microsoft 365 Lighthouse API", which describes intent better than the noun version.

I also added "in Microsoft Graph" to the description.

File: concepts/upload-data-to-upload-session.md [hunk 5]
```diff
@@ -136,8 +137,9 @@ Content-Type: application/json
   ]
 }
 ```
-## Code examples: Create upload session and upload documents
- 
+
+## Create upload session and upload documents: code examples
```
[the remark this answers is not in the document]
Comment (Lauragra) [reply concepts/upload-data-to-upload-session.md]: I tried to fix the structure of the topic the way I think that it should be. I think we want to lead with code examples in that particular heading; it's giving the SDK snippets for the previous operations in a couple of languages.

**Review by Lauragra — APPROVED** [review]

File: concepts/upload-data-to-upload-session.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Linda-Editor) [reply concepts/upload-data-to-upload-session.md]: I like your fixes. Thanks!

**Pull request merged**